lib/que/worker.rb

```diff
@@ -237,7 +237,7 @@ def work
           :job_worked
         end
       end
-    rescue PG::Error => _error
+    rescue PG::Error, Adapters::UnavailableConnection => _error
```

Comment:
I have a feeling we can just catch `::ActiveRecord::ConnectionTimeoutError` here as any other adapter is fundamentally unmaintained...

Reply:
Seems fine to have the specific error.